### DISCUSSION TITLE
关于复旦大学陈阳老师团队的连接

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#355] [1️⃣ 新鲜事]
近期通过《[基于机器学习的GitHub企业影响力分析与预测](https://www.infocomm-journal.com/znkx/CN/10.11959/j.issn.2096-6652.202327)》一文，连接到复旦大学的[陈阳老师团队](https://chenyang03.wordpress.com/)，和咱们实验室的工作非常相关与互补，陈老师也在[上海开源信息技术协会](https://shanghaiopen.org.cn/)下面的一个专委会开展一些工作。

几个亮点包括：
 - 自主开源项目 [EasyGraph](https://easy-graph.github.io/)，可以和 OpenDigger 进行结合；
 - [开源社区开发者行为分析](https://github.com/chenyang03/developers)
 - 结构洞方面的研究（联合清华大学[罗家德](https://www.sss.tsinghua.edu.cn/info/1043/3910.htm)）

后续各种活动可以积极协作起来~ :-)

**Comment by will-ww** [comment]
特别是 EasyGraph，一个很不错的开源网络分析工具：[EasyGraph.pdf](https://github.com/X-lab2017/open-wonderland/files/14118529/EasyGraph.pdf)

<div align=center>
<img src="https://github.com/X-lab2017/open-wonderland/assets/15010826/a5a94d8a-075a-40e7-ad4e-40b7cb4d89c9" width="650px">
</div>

**Comment by chenyang03** [comment]
感谢王老师！欢迎X-LAB师生多多指导我们的工作！
